docs/restore-individual-tables.md

```diff
@@ -1,41 +1,50 @@
 # Restore individual tables
 
-Percona XtraBackup can export individual tables from any InnoDB database, if they are individual `.ibd` files, and import them into Percona Server for MySQL with XtraDB or MySQL 8.0. The source doesn’t have to be XtraDB or MySQL 8.0, but the destination must be.
+You can use Percona XtraBackup to move specific tables from one database to another. This works for InnoDB tables that are stored as separate `.ibd` files. You can move these tables into either Percona Server for MySQL with XtraDB or MySQL 8.0. Your source database can be any type, but your destination must be XtraDB or MySQL 8.0.
```

Comment:
IMO, we should move this entire document/page to "How to" section. Or atelast add a duplicate reference.